python/numpy-f2py-compile.yml

```diff
@@ -0,0 +1,11 @@
+rules:
+- id: numpy-f2py-compile
+  patterns:
+    - pattern-either: 
+      - pattern: |
+          np.f2py.compile(...)
```

Comment:
I would turn "np" into "numpy" to generalize across different forms of the library name.